composer.json

```diff
@@ -10,7 +10,12 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^5.0 | ^6.0",
-        "zendframework/zend-mvc": "^2.0 | ^3.0"
+        "zendframework/zend-mvc": "^2.0 | ^3.0",
+        "zendframework/zend-view": "^2.0",
```

Comment:
I added `zend-mvc` a week back. So its fine if there are more of them to execute tests properly